middlewares/hacker.middleware.js

```diff
@@ -55,7 +55,7 @@ function parseHacker(req, res, next) {
     delete req.body.needsBus;
     delete req.body.application;
     delete req.body.authorization;
-    
+
```

Comment:
Remove change

Reply:
I think this is from the auto-formating.